src/app/components/verse/verse.component.ts

```diff
@@ -22,21 +22,29 @@ export class VerseComponent {
     return this.evtModelService.lines$.pipe(
       map(lines => lines.length > 0),
       map(hasLines => {
-        // In diplomatic and interpretative edition, if the text doesn't have any line, verses are shown as block items
-        // In critical edition verses are always shown as block items
+        // In diplomatic and interpretative edition, if the text doesn't have any line, verses are shown as block items,
+        // unless current text flow is prose
+        // In critical edition verses are always shown as block items, unless current text flow is prose
         switch (this.editionLevel) {
           case 'diplomatic':
           case 'interpretative':
-            return !hasLines;
+            return this.textFlow === 'verses' ? true : !hasLines;
           case 'critical':
-            return true;
+            return this.textFlow === 'prose' ? false : true;
         }
       }),
     );
   }
 
+  get showNumber() {
+    const num = parseInt(this.data.n, 10);
+
+    return !isNaN(num) && num % 5 !== 0;
```

Comment:
Should `5` become a parameter or, at least, a constant of the component?

Reply:
We never thought about it. Probably it would be nice to make it a configurable parameter